apps/kfp-tekton/upstream/base/installs/multi-user/pipelines-profile-controller/kustomization.yaml

```diff
@@ -13,4 +13,4 @@ configMapGenerator:
   - sync.py
 - name: kubeflow-pipelines-profile-controller-env
   envs:
-  - params.env
+  - params.env
```

Comment:
No newline at the EOF

Reply:
Done 👍🏾

Reply:
please check the whole MR in Github yourself.

Reply:
Finished; is there a code style doc for this repo I can reference for standards like this?

Reply:
This is a git error not codestyle
https://thoughtbot.com/blog/no-newline-at-end-of-file